lgc/builder/SubgroupBuilder.cpp

```diff
@@ -755,10 +771,12 @@ Value *SubgroupBuilder::CreateSubgroupClusteredInclusive(GroupArithOp groupArith
 //
 // @param groupArithOp : The group arithmetic operation.
 // @param value : An LLVM value.
-// @param clusterSize : The cluster size.
+// @param ctSize : The cluster size.
```

Comment:
ctSize -> expectedClusterSize
The same as above

Reply:
expectedClusterSize is a bit long, can i rename it to the inClusterSize.